podman_compose.py

```diff
@@ -2870,38 +2904,44 @@ async def compose_down(compose: PodmanCompose, args):
             timeout = str_to_seconds(timeout_str)
         if timeout is not None:
             podman_stop_args.extend(["-t", str(timeout)])
-        down_tasks.append(
-            asyncio.create_task(
-                compose.podman.run([], "stop", [*podman_stop_args, cnt["name"]]), name=cnt["name"]
-            )
-        )
-    await asyncio.gather(*down_tasks)
+        await compose.podman.run([], "stop", [*podman_stop_args, cnt["name"]])
```

Comment:
This will way on all services to stop one by one. This is problematic in case of large number of services. One deployment I have is 40 services for example.

Reply:
They would stop in parallel as await yields from the `compose.podman.run([], "stop", [*podman_stop_args, cnt["name"]])`.
So rather than waiting for `asyncio.gather` to schedule the future tasks/co-routines, here they get scheduled as soon as they are passed.

However, the concern is valid. It won't wait for all the stop tasks to finish before proceeding to `await compose.podman.run([], "rm", [cnt["name"]])`. I'll try to fix it. (Ideally should be upgraded to TaskGroup, but for that there has to be an upgrade mandate to Python 3.11 for compatibility reasons).

Reply:
Done. Updated the code to use `asyncio.gather` for `stop` and `rm` command tasks.